csi/src/node/nvmfutil.rs

```diff
@@ -0,0 +1,179 @@
+use std::{io, thread, time};
+use udev::Enumerator;
+
+fn find_nvmf_device_by_uuid(str_uuid: &str) -> Result<String, String> {
+    let prop = "ID_WWN";
+    let value = format!("uuid.{}", str_uuid);
+
+    trace!("find_nvmf_device_by_uuid uuid={}", str_uuid);
+    let mut enumerator = Enumerator::new().unwrap();
+    enumerator.match_subsystem("block").unwrap();
+    //    enumerator.match_property("DEVTYPE", "disk").unwrap();
```

Comment:
Should the commented out line be removed?

Reply:
https://github.com/openebs/Mayastor/pull/302/commits/103a309bb8f0d25e4b4cf62544a0d24d400cd998